app/assets/stylesheets/_notes.scss

```diff
@@ -1,11 +1,16 @@
+notes-view {
+  width: 350px;
+}
+
```

Comment:
Should we use a class `.notes-view` here?

I really like how angular let you define new html element, but we'd lose the style without really knowing why when we migrate the parent component to react.